sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt

```diff
@@ -508,4 +508,8 @@ internal class NativeMapImpl(private val map: MapInterface) :
   override fun unsubscribe(observer: Observer) {
     map.unsubscribe(observer)
   }
+
+  override fun setMapProjection(value: Value) = map.setMapProjection(value)
+
+  override fun getMapProjection() = map.mapProjection
```

Comment:
Don't we exposes these two API in this PR yet?

Reply:
no, as mentioned in description, work will be done in https://github.com/mapbox/mapbox-maps-android/pull/667